inspirehep/modules/records/api.py

```diff
@@ -71,6 +74,13 @@ def delete(self):
     def _delete(self, *args, **kwargs):
         super(InspireRecord, self).delete(*args, **kwargs)
 
+    def commit(self, validator=InspireValidator):
```

Comment:
Do we need to pass the `validator` at `commit`-time? Can it be done directly at constructor? Is there a time where we do not want to use `InspireValidator` and hence it's desirable to call `commit(validator=None)`?